### PR TITLE
Removes some references to /datum/reagent/colorful_reagent/powder's old name that Akrilla forgot about

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

This PR actually renames the crayon powder chem to "colorful powder", instead of just changing its path and calling it a day.

## Why It's Good For The Game

I don't agree with the merging of Akrilla's PR (https://github.com/tgstation/tgstation/pull/46210), but I can at least fix some minor issues with it.

## Changelog
:cl:
tweak: Changed the default name of /datum/reagent/colorful_reagent/powder to "mundane powder" (from "crayon powder"), to better reflect that it can't color anything (its subtypes still can, of course) and that /datum/reagent/colorful_reagent/powder isn't found in most crayons anymore.
tweak: Changed the names of the subtypes of /datum/reagent/colorful_reagent/powder so that they no longer mention crayons/"crayon powder".
tweak: Changes the wordings of the descriptions of /datum/reagent/colorful_reagent/powder and its subtypes to be less weird (and to mention that invisible powder and mundane powder can't color anything). This change should also make the description of mundane powder no longer state that it can color things the "none" color.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
Note to self: once the antag PR freeze is over, add a "real" invisible crayon item for mime traitors that can actually color stuff invisible.

**Comment by Fhaxaris** [comment]
ATH gang ATH gang

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1216,72 +1216,77 @@
...
 	taste_description = "the back of class"
 
 /datum/reagent/colorful_reagent/powder/New()
-	description = "\an [colorname] powder good for colouring chemical reagents."
+	if(colorname == "none")
+		description = "A rather mundane-looking powder. It doesn't look like it'd color much of anything..."
+	else if(colorname == "invisible")
+		description = "An invisible powder. Unfortunately, since it's invisible, it doesn't look like it'd color much of anything..."
+	else
+		description = "\an [colorname] powder, used for coloring things [colorname]."
```
Comment (SpaceManiac) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: probably wrong capitalization?

Comment (ATH1909) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: @SpaceManiac @ExcessiveUseOfCobblestone change done

**Comment by ExcessiveUseOfCobblestone** [comment]
If I comment on something I get notifs when you update it so you don't HAVE to ping.

**Pull request merged**